src/main/java/frc/team2412/robot/Robot.java

```diff
@@ -178,7 +237,7 @@ public void autonomousInit() {
             subsystems.drivebaseSubsystem.resetPose(RigidTransform2.ZERO);
         }
 
-        if (SubsystemConstants.SHOOTER_ENABLED) {
+        if (SubsystemConstants.SHOOTER_ENABLED && isCompetition()) {
```

Comment:
Shouldn't need this, as if shooter is enabled on the test bot, we can assume it has encoders too.